connectivity/check/action.go

```diff
@@ -510,197 +549,221 @@ func (a *Action) GetIngressRequirements(p FlowParameters) []filters.FlowSetRequi
 	return []filters.FlowSetRequirement{ingress}
 }
 
-var errNeedMoreFlows = errors.New("Required flows not found yet")
+// getHubbleStatus polls the server status from Relay until either all the Hubble instances are
+// available (success) or the context is cancelled (failure).
+func (a *Action) getHubbleStatus(ctx context.Context, client observer.ObserverClient) error {
```

Comment:
`getHubbleStatus` is a poor choice of name given what the method actually does. It's probably also worth adding a comment that this check does not make sense if not targeting Hubble Relay.